packages/config/src/handlers/application/applicationConfigHandler.ts

```diff
@@ -289,5 +298,6 @@ const applicationConfigResponseValidation = Joi.object({
   INFORMANT_NOTIFICATION_DELIVERY_METHOD: Joi.string().allow('').optional(),
   SIGNATURE_REQUIRED_FOR_ROLES: Joi.array().items(
     Joi.string().valid(...SystemRoleType)
-  )
+  ),
+  SEARCH_DEFAULT_CRITERIA: Joi.string().valid(...searchCriteria)
```

Comment:
```suggestion
  SEARCH_DEFAULT_CRITERIA: Joi.string().valid(...searchCriteria).optional()
```

Reply:
It's optional by default. But for better readability and to keep it consistent with the other payload properties, adding it here @Zangetsu101